providers/imap/src/airflow/providers/imap/hooks/imap.py

```diff
@@ -85,6 +85,7 @@ def get_conn(self) -> ImapHook:
 
     def _build_client(self, conn: Connection) -> imaplib.IMAP4_SSL | imaplib.IMAP4:
         mail_client: imaplib.IMAP4_SSL | imaplib.IMAP4
+        host = conn.host if conn.host else ""
```

Comment:
This can be a bit simplified like:
```suggestion
        host = conn.host or ""
```

Reply:
Thanks @jscheffl. I'll fix it real quick!